package.json

```diff
@@ -26,14 +26,16 @@
   },
   "types": "./typings/index.d.ts",
   "dependencies": {
-    "@types/jquery": "^2.0.34",
-    "@types/jqueryui": "^1.11.31",
-    "@types/knockout": "^3.4.38",
+    "@types/jquery": ">=2.0.34",
+    "@types/jqueryui": ">=1.11.31",
+    "@types/knockout": ">=3.4.38",
+    "@types/knockout-secure-binding": ">=0.0.28",
+    "@types/mousetrap": ">=1.5.0",
```

Comment:
Are you referring to mousetrap? If so, yeah, not much of a choice. Would love to not have this dependency though...